app.js

```diff
@@ -107,3 +106,27 @@ var makeDisShit2 = function(){
 makeDisShit2();
 
 
+//==================================
+var makeNewStore = function(makeStore){
+  makeStore.preventDefault();
+  makeStore.stopPropagation();
+  var lolol = document.getElementById('tfooti');
+  lolol.parentElement.removeChild(lolol);
+
+  var name = makeStore.target["store-name"].value;
+  var minCust = makeStore.target["store-min"].value;
+  var maxCust = makeStore.target["store-max"].value;
+  var avgCook = parseInt(makeStore.target["store-avg"].value);
+  var openHours = makeStore.target["store-hours"].value;
+  console.log(name, minCust, maxCust, avgCook); 
+  var objName = new Store(name, minCust, maxCust, avgCook, openHours, [], 0 );
+  storesArray.push(objName);
+  objName.makeGraphRow();
+  makeTotalsRow();
+  console.log(makeNewStore);

```

Comment:
line 126 here. what do you think this console.log should give you?
what does it give you?